src/lib/fakeCozyClient.js

```diff
@@ -0,0 +1,13 @@
+export const createFakeCozyClient = (uri, token) => {
+  return {
+    isFakeCozyClient: true,
+    stackClient: {
+      uri,
+      token: {
+        token
```

Comment:
c'est pas accessToken ici ?

Reply:
['cozy-realtime' test les 2](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-realtime/src/index.js#L85), je ne sais jamais lequel il faut mettre...

Reply:
C'est accessToken.

Reply:
Hum, en fait dans un AppToken c'est bien `token.token`. Le `token.accessToken` est pour Oauth seulement. Oui, ce n'est pas idéal d'avoir deux chemins distincts pour ce qui est un seul et même token…
Si on n'a pas de refreshToken je dirais qu'il vaut mieux rester dans le cas le plus simple et donc rester sur un AppToken

Reply:
https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/AppToken.js

Reply:
Solved since we now create the client :)